central/graphql/resolvers/node_vulnerabilities.go

```diff
@@ -80,8 +82,366 @@ func (resolver *Resolver) NodeVulnCounter(ctx context.Context, args RawQuery) (*
 		query := withNodeCveTypeFiltering(args.String())
 		return resolver.vulnCounterV2(ctx, RawQuery{Query: &query})
 	}
-	// TODO : Add postgres support
-	return nil, errors.New("Resolver NodeVulnCounter does not support postgres yet")
+	return resolver.nodeCveCounterV2(ctx, args)
+}
+
+func (resolver *Resolver) nodeCveV2(ctx context.Context, args IDQuery) (NodeVulnerabilityResolver, error) {
+	if err := readCVEs(ctx); err != nil {
```

Comment:
Check node permission. All parts of nodes should rely on users having permission to node (similarly for images @keyallis)

Reply:
Yes, node components and node vulns should use node permission.